### PR TITLE
Update to latest dace version

### PR DESCRIPTION
**Pull request by orausch**

### CODE REVIEW

File: daceml/onnx/op_implementations/pure_implementations.py [hunk 1]
```diff
@@ -218,87 +221,78 @@ def forward_can_be_applied(node: ONNXOp, state: SDFGState,
...
-                    '_c':
-                    dace.Memlet.simple(
-                        "Y", '__i0, __i1', wcr_str='lambda x, y: x + y')
-                },
-                external_edges=True)
-            return sdfg_exp
+        input0_dim = in_desc_with_name(node, state, sdfg, "A").shape
+        input1_dim = in_desc_with_name(node, state, sdfg, "B").shape
+
+        # list containing letters from a-z
```
Comment (tbennun) [inline daceml/onnx/op_implementations/pure_implementations.py:229]: z-a

File: setup.py [hunk 2]
```diff
@@ -23,8 +23,8 @@
     packages=['daceml'],
     package_data={'': ['*.cpp']},
     install_requires=[
-        'dace@git+https://github.com/spcl/dace.git@d3571a8', 'onnx == 1.7.0',
-        'torch'
+        'dace@git+https://github.com/orausch/dace.git@daceml_branch',
```
Comment (tbennun) [inline setup.py:26]: sad. This is only because of the serialization issue?

**Review by tbennun — APPROVED** [review]
Looks good, left a question as a comment. I don't know why coverage dropped so much.

File: setup.py (hunk 2 above)
Comment (orausch) [reply setup.py:26]: Yes, at the moment that's the blocker

**Pull request merged**